{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/deployment/model_deployment/deploy.py

```diff
@@ -1,6 +1,9 @@
+import sys
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.parent.resolve()))
```

Comment:
The reason we added and moved the imports above this line is because Python wasn't able to discover and import from `utils`?

Reply:
Yeah. I tried several ways. 
For CUJ project, the "current directory" will be the outer "mlops-azure-cuj" folder. The best way to import utils will be `mlops-azure-cuj.utils`. However `mlops-azure-cuj` is not a valid package name and it doesn't have to be. 
So I used absolute path to add the location of utils to `sys.path` and directly imported it.